src/js/components/Button/StyledButton.js

```diff
@@ -153,13 +153,11 @@ const StyledButton = styled.button`
     props.focus && (!props.plain || props.focusIndicator) && focusStyle}
   ${props =>
     !props.plain &&
+    props.theme.button.transition &&
     `
-    transition-property: color,
-      background-color,
-      border-color,
-      box-shadow;
-    transition-duration: 0.1s;
-    transition-timing-function: ease-in-out;
+    transition-property: ${props.theme.button.transition.properties.join(', ')};
```

Comment:
```suggestion
    transition-property: ${props.theme.button.transition.properties.join(',')};
```
Removing the space since it doesn't have any affect.